testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/account/AccountRestServiceTest.java

```diff
@@ -963,7 +963,9 @@ public void listApplicationsThirdParty() throws Exception {
         assertFalse(app.getConsent().getGrantedScopes().isEmpty());
         ConsentScopeRepresentation grantedScope = app.getConsent().getGrantedScopes().get(0);
         assertEquals(clientScopeRepresentation.getId(), grantedScope.getId());
-        assertEquals(clientScopeRepresentation.getAttributes().get(ClientScopeModel.CONSENT_SCREEN_TEXT) != null ? clientScopeRepresentation.getAttributes().get(ClientScopeModel.CONSENT_SCREEN_TEXT) : clientScopeRepresentation.getName(), grantedScope.getName());
+
+        String consentScreenText = clientScopeRepresentation.getAttributes().get(ClientScopeModel.CONSENT_SCREEN_TEXT);
+        assertEquals(consentScreenText != null && !consentScreenText.isEmpty() ? consentScreenText : clientScopeRepresentation.getName(), grantedScope.getName());
```

Comment:
This is weird, as why would the consentScreenText be null sometimes, and sometimes not?